packages/fast-usdc-contract/test/supports.ts

```diff
@@ -184,29 +190,61 @@ export const commonSetup = async (t: ExecutionContext<any>) => {
 
   await registerKnownChains(agoricNamesAdmin, () => {});
 
+  // XXX: copied from orchestration supports
   let ibcSequenceNonce = 0n;
-  /** simulate incoming message as if the transfer completed over IBC */
-  const transmitTransferAck = async () => {
+  const transmitVTransferEvent = async (
```

Comment:
I wonder if this dual-maintenance warrants exporting this via `tools` in `@agoric/orchestration`